src/patterns/dosdp-patterns/NSForestMarkers.yaml

```diff
@@ -0,0 +1,57 @@
+pattern_name: NSForestMarkers
+pattern_iri: http://purl.obolibrary.org/obo/cl/NSForestMarkers.yaml
+description: "Characterizing marker sets template for cell terms."
+
+classes:
+  "thing": "owl:Thing"
+  "sequence_feature": "SO:0000110"
+
+relations:
+  has_part: "BFO:0000051"
+#  has_anatomical_context: "PCL:0010063"
+
+vars:
+  Organ_region: "'regional part of brain'"
+  Parent: "'thing'"
+
+list_vars:
+  Minimal_markers: "'sequence_feature'"
+
+data_vars:
+  Marker_set_of: "xsd:string"
+  Organ: "xsd:string"
+  Species_abbv: "xsd:string"
+  FBeta_confidence_score: "xsd:double"
+
+annotationProperties:
+  fbetaConfidenceScore: "PCL:0010062"
```

Comment:
Recently STATO added a new term for F Beta Score https://github.com/ISA-tools/stato/pull/86#issuecomment-2221606864.

Should we use [STATO:0000663](http://purl.obolibrary.org/obo/STATO_0000663)(F-beta score) instead of `fbetaConfidenceScore: "PCL:0010062"`

Reply:
yes please